README.md

```diff
@@ -0,0 +1,61 @@
+# About workshop
+
+In this workshop we will explore what Firebase Functions are and setup project to be able to write those functions in Kotlin. To get there we will touch some other interesting things like Realtime Database and TypeScript. 
+
+
+### Requirements:
```

Comment:
Will these be set up during the WS or do they have to be prepared? If it's the latter, `Prerequisites` might be better suited here.